TOC-tidb-cloud.md

```diff
@@ -75,6 +75,7 @@
   - Plan Your Cluster
     - [Select Your Cluster Tier](/tidb-cloud/select-cluster-tier.md)
     - [Determine Your TiDB Size](/tidb-cloud/size-your-cluster.md)
+    - [TiDB Cloud Performance Reference](/tidb-cloud/tidb-cloud-performance-reference.md)
```

Comment:
```suggestion
    - [TiDB Cloud Performance Test Report](/tidb-cloud/tidb-cloud-performance-reference.md)
```
or
```suggestion
    - [TiDB Cloud Performance Benchmarks](/tidb-cloud/tidb-cloud-performance-reference.md)
```
or
```suggestion
    - [TiDB Cloud Benchmarks](/tidb-cloud/tidb-cloud-performance-reference.md)
```

Reply:
After checking with QA, the current doc does not contain any test steps, so it cannot be defined as a test result or benchmark docs.